hswindow/hsdialog/dialog.go

```diff
@@ -32,9 +30,14 @@ func (d *Dialog) Show() {
 	d.Visible = true
 }
 
-// RegisterKeyboardShortcuts registers a new shortcut
-func (d *Dialog) RegisterKeyboardShortcuts(_ *hsinput.InputManager) {
-	// noop
+// RegisterKeyboardShortcuts registers a shortcuts for popup dialog
+func (d *Dialog) RegisterKeyboardShortcuts(_ ...giu.WindowShortcut) {
+	// https://github.com/OpenDiablo2/HellSpawner/issues/327
+}
+
+// KeyboardShortcuts returns a list of shortcuts
+func (d *Dialog) KeyboardShortcuts() []giu.WindowShortcut {
+	return []giu.WindowShortcut{}
 }
```

Comment:
If we wanted the dialog modal to close when pressing escape, this is where we would define that, right?

Reply:
no, if we'd like to add shortcut to some specified dialog we need to handle it here. Closing dialogs by escape is currently done here: https://github.com/OpenDiablo2/HellSpawner/blob/2a0e033f27d6296bda1b1714b8ee5cef05d5e672/hsapp/setup.go#L260